jsk_robot_common/jsk_robot_startup/lifelog/active_user.l

```diff
@@ -71,6 +71,34 @@
   (update-activeness)
   )
 
+(defun get-username ()
+  (let (username)
+    (unix::system
+     (format nil
+             "whoami > /tmp/username_~d.txt"
+             (unix::getpid)))
+    (warn "Get PID ~A" (unix::getpid))
+    (with-open-file
+        (f (format nil "/tmp/username_~d.txt" (unix::getpid)))
+      (setq username (read-line f)))
+    username
```

Comment:
I think we should remove this generated file, `(unix::unlink "/tmp/username...")` ??  or use `(defun get-username () lisp::*user*)`